lib/processLayerStyles.js

```diff
@@ -72,12 +75,62 @@ const layerFill = (layerStyle) => {
 
         }
 
+        if (fill.fillType == 4 && fill.isEnabled == true) { // Images
+          // Copy the image from the sketch file to the output folder
+          let imageOutputFolder = outputFolder + "/assets/Assets.xcassets"
+          let imageName = fill.image._ref.split("/").splice(-1)[0]
```

Comment:
Can this be explained? I _think_ it's saying take the ref, break it down using "/", remove the last element and then take the first - but that doesn't make any sense?

Reply:
Almost -- splice(-1) removes all items from the array except the last one, so you end up with [lastElement] and then take the first one which just so happens to be the only one (`[0]`)

Probably simpler/more undestandable as `fill.image._ref.split("/").pop()`?

Reply:
Ah, I'd assumed (and thought I'd tested but obviously hadn't correctly!) that it took off the last element, not returned only the last element! Shame there isn't a `last` method on arrays :)

FWIW I much much prefer `pop()`